draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1091,68 +1093,61 @@ Response with ciphertext = 0x60b035059d9ef5667c5a0710823b and no Partial IV.
+Group OSCORE allows to preserve a long exchange active indefinitely, even in case the group is rekeyed, with consequent change of ID Context, or in case the client obtains a new Sender ID.
 
-Upon leaving the group or before re-joining the group, a group member MUST terminate all the ongoing Non-Notification Group Exchanges and observations that it has started in the group as a client, and hence frees up the CoAP Token associated with the corresponding request.
+As defined in {{mess-processing}}, this is achieved by the client and server(s) storing the 'kid' and 'kid context' used in the original request, throughout the whole duration of the long exchange.
 
+Upon leaving the group or before re-joining the group, a group member MUST terminate all the ongoing long exchanges that it has started in the group as a client, and hence frees up the CoAP Token associated with the corresponding request.
 
 ## Synchronization and Freshness # {#sec-freshness}
 
 If the application requires freshness, e.g., according to time- or event-based policies (see {{Section 2.5.1 of RFC9175}}), a server can use the approach in {{sec-synch-challenge-response}} as a variant of the procedure in {{Section B.1.2 of RFC8613}}, before delivering request messages from a client to the application. This also makes the server (re-)synchronize with the client's Sender Sequence Number.
 
-Like in OSCORE {{RFC8613}}, assuming an honest server, the message binding guarantees that a response is not older than the request it replies to, so the same properties as stated in {{Section 7.3 of RFC8613}} apply:
+Like in OSCORE {{RFC8613}}, assuming an honest server, the message binding guarantees that a response is not older than the request it replies to. Therefore, building on {{Section 7.3 of RFC8613}}, the following properties hold for Group OSCORE.
 
 * The freshness of a response can be assessed if it is received soon after the request.
 
-* For notifications, this assessment gets weaker with time, and it is RECOMMENDED that the client regularly re-register the observation.
+   For responses within a long exchange, this assessment gets weaker with time. If such responses are Observe notifications {{RFC7641}}, it is RECOMMENDED that the client regularly re-register the observation.
 
-* It is not guaranteed that a misbehaving server did not create the response before receiving the request, i.e., Group OSCORE does not verify server aliveness.
-
-* For requests and notifications, the received Partial IV allows a recipient to determine the relative order of requests or responses.
-
-This applies also to non-notification responses:
+   If the request was neither a group request nor an Observe request, there is at most a single response and only from one, individually targeted server in the group. Thus, freshness can be assessed depending on when the request was sent.
```

Comment:
Considering the discovery method described in the end of "Message Processing in Pairwise Mode", and in "Pre-Conditions", can it make more sense to say one of the following?
_there is at most a single valid response and only from one_
or
_there is at most a single protected response and only from one_

Reply:
Yes, it's definitely only about a valid response. The first option looks good to use.

`there is at most a single valid response and only from one ...`